### PR TITLE
Federation Experience Frontend - View

### PR DESCRIPTION
**Pull request by sestuna117**
Signed-off-by: Jason-Guo <jason.guo@ibm.com>

### CODE REVIEW

File: frontend/src/components/federation-list.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
+import { Component } from 'react';
+import { connect } from 'react-redux';
+import IsManager from './is_manager';
```
Comment (mamy-CS) [inline frontend/src/components/federation-list.tsx:3]: Remove import since it is not used

File: frontend/src/tables/federations-list-table.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import { connect } from 'react-redux';
+import IsManager from 'components/is_manager';
```
Comment (mamy-CS) [inline frontend/src/tables/federations-list-table.tsx:3]: Remove unused import here as well

**Review by mamy-CS — COMMENTED** [review]
And let's add an Info column, same as "Agents list" table where we dump the federation relationship info in a json format. See here for an example: https://github.com/spiffe/tornjak/blob/71a03a41a87f1ba5af591a811bc6f4bdb68a8a94/frontend/src/tables/agents-list-table.tsx#L77
Screenshot 2024-10-21 at 1 11 15 PM

**Comment by sestuna117** [comment]
Suggested changes have been made and pushed
- removed unused imports
- moved location on navbar
- included Info column in table

**Review by mamy-CS — APPROVED** [review]
/lgtm thank you for your contribution @sestuna117

**Pull request merged**